### DISCUSSION TITLE
Seemingly cannot disable ligature for backticks/ backquotes/code fences/code blocks?

### DISCUSSION THREAD

**Discussion by SichangHe** [kovidgoyal/kitty#7279] [Q&A] [answered]
![image](https://github.com/kovidgoyal/kitty/assets/84777573/438cbdcd-88d4-4401-bc3e-df3125619688)

Notice that the ` `` ` and ` ``` ` are almost invisible because of what seems to be JetBrains Mono Nerd Font's ligature.

Although, I disabled the ligature, as show in the `=>` and `->` above:

```
#  kitty/kitty.conf
disable_ligatures always
```

Anyone has any ideas, please?

**Comment by kovidgoyal** [comment]
1) Dont use patched fonts, https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font

2) Disabling ligatures only disables calt based ligatures, if your font uses some other type of ligature it wont have any effect.

**Reply by SichangHe** [reply, marked as answer]
Thanks a lot for the clarification on the font!

Solution for me is to use a font without ligature:

```
# kitty/kitty.conf
font_family      JetBrains Mono NL
```
